person.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
```

Comment:
- [ ] The same thing here, inside the `person` class, we need to initialize the rentals, as `@rentals = []`. Also once you add the rentals to the initializer please create `attr_accessor` to it. We need to be able to check the rentals for the person. 
- [ ] Also, when we add a rental we need to create a new rental with the `person` as a self and add the newly created rental to the rentals list, This way the association will be preserved all the way. 

> The method should look something like...
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```